lib/index.js

```diff
@@ -15,16 +16,32 @@ export const metadata = {
   version: require('../package.json').version
 };
 
+export const mapComponentDispatch = {
+  Header: (dispatch, map) =>
+    Object.assign(map, {
+      setCurrentClient: info => dispatch(setCurrentClient(info))
+    })
+};
+
 export const mapComponentState = {
   Header: (state, map) =>
     Object.assign(map, {
       bcoinUri: state.node.serverInfo.bcoinUri,
```

Comment:
Maybe this is out of scope of this PR, but this should probably change

Reply:
Yeah, I was trying to rip this out actually from this plugin. Just missed this one. I’m actually wondering if it’s worth taking out this endpoint entirely now.